test-case/check-reboot.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+##
+## Case Name: test reboot
+## Preconditions:
+##    N/A
+## Description:
+##    run reboot for the test
+## Case step:
+##    1. check system status is correct
+##    2. wait for the delay time
+##    3. trigger for the reboot
+## Expect result:
+##    Test execute without report error in the LOG
+##
+
+source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+
+OPT_OPT_lst['l']='loop'     OPT_DESC_lst['l']='loop count'
+OPT_PARM_lst['l']=1         OPT_VALUE_lst['l']=3
+
+OPT_OPT_lst['d']='delay'    OPT_DESC_lst['d']='delay after system bootup'
+OPT_PARM_lst['d']=1         OPT_VALUE_lst['d']=2
+
+func_opt_parse_option $*
+
+func_lib_check_sudo
+loop_count=${OPT_VALUE_lst['l']}
+delay=${OPT_VALUE_lst['d']}
+
+# write the total & current count to the status file
+status_log=$LOG_ROOT/status.txt
```

Comment:
When booting failed, it may boot to generic kernel. verify_list will fail with generic but should we remember kernel name/version being tested against?

Reply:
thats definitely a good point.

Reply:
check kernel version is good idea

Reply:
Maybe we should keep a list for the verify test cases to compare if the env is changed.

Reply:
How about add more environment into [sof-dump-status.py](https://github.com/thesofproject/sof-test/blob/master/tools/sof-dump-status.py)  

so here direct load this script store into the file & compare the file content